css/style.css

```diff
@@ -0,0 +1,17 @@
+body {
+  background-color: grey;
+  text-align: center;
+}
+
+#image {
+  width: 100%;
+  height: auto;
+  max-width: 625px;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+}
```

Comment:
Kindly style your image again and make it to be similar with the original tribute page.
 Here in your project the image size is smaller and didn't cover the whole page just like in there.